Formula/verapdf-apps.rb

```diff
@@ -0,0 +1,27 @@
+class VerapdfApps < Formula
+  desc "Open-source industry-supported PDF/A validation"
+  homepage "https://verapdf.org/home/"
+  url "https://github.com/veraPDF/veraPDF-apps/archive/refs/tags/v1.21.4.tar.gz"
+  sha256 "b926c397f98c03c506e486593e0224f4250c5329ed353a280e4dcfa26ca09b68"
+  license any_of: ["GPL-3.0-or-later", "MPL-2.0"]
+  head "https://github.com/veraPDF/veraPDF-apps.git", branch: "integration"
+
+  depends_on "maven" => :build
+  depends_on "openjdk" => :build
```

Comment:
```suggestion
  depends_on "openjdk"
```
Looks like you need this at runtime.